numcodecs/compat.py

```diff
@@ -1,9 +1,7 @@
 # flake8: noqa
```

Comment:
```suggestion
```
If you get rid of this line, `ruff` should run properly on this file. I tried it locally and it seems to work.

Reply:
This is handled in a different PR: #559 / https://github.com/zarr-developers/numcodecs/pull/559/commits/1c980e5f7ebb12ce9da07ed1aa08e8ff88fad5a4.

I had to silence `F401` and `E402` in preview mode.